lib/src/restartable_timer.dart

```diff
@@ -44,11 +44,11 @@ class RestartableTimer implements Timer {
     _timer.cancel();
   }
 
+  /// The number of durations preceding the most recent timer event on the most
+  /// recent countdown.
+  ///
+  /// Calls to [reset] will also reset the tick so subsequent tick values may
+  /// not be strictly larger than previous values.
   @override
```

Comment:
Do remove the `@override` while you are here. We don't use `@override` in `package:async`. (This is the only instance of `@override` in the entire package).

Reply:
Sure, I can for now.

I think we should strongly consider adding them everywhere in this package and enabling the lint. We implement a lot of interfaces and it's really nice to be able to see quickly which are the new methods.